docs/CONTRIBUTING.md

```diff
@@ -22,25 +38,25 @@ and then open up a PR, or "Pull Request," in the upstream HJR repository. Again,
 
 After you have setup a PR, it will be [reviewed](#reviewing-a-pr) by maintainers and changes may be requested. Make the changes requested and eventually it will likely be accepted and merged into main.
 
-## Core Principles
+## Core Principles<a name="core-principles"></a>
 
 In creating HJR, we had a few principles in mind for development:
 
 1. Minimize the number of options written;
-2. Include only the module collection - leave functionality to Hjem; and
-3. Maintain readability of code, even for new users.
+1. Include only the module collection - leave functionality to Hjem; and
+1. Maintain readability of code, even for new users.
```

Comment:
This is to reduce diffs when inserting new items into the list, GitHub for instance renders the Markdown as having incrementing nnumbers.

Reply:
Is that automatic from `mdformat-gfm`? I'm fine with that, not a fan of the semantic changes but that's not blocking.

Reply:
Yup, only ran mdformat over this file